tests/integrationv2/test_signature_algorithms.py

```diff
@@ -7,25 +7,21 @@
 from providers import Provider, S2N, OpenSSL, GnuTLS
 from utils import invalid_test_parameters, get_parameter_name, get_expected_s2n_version, to_bytes
 
-certs = [
-    Certificates.RSA_2048_SHA256,
-    Certificates.RSA_2048_SHA384,
-    Certificates.RSA_PSS_2048_SHA256,
-    Certificates.ECDSA_256,
-    Certificates.ECDSA_384,
-]
-
 all_sigs = [
     Signatures.RSA_SHA1,
     Signatures.RSA_SHA224,
     Signatures.RSA_SHA256,
     Signatures.RSA_SHA384,
     Signatures.RSA_SHA512,
     Signatures.ECDSA_SECP256r1_SHA256,
+    Signatures.ECDSA_SECP521r1_SHA512,
```

Comment:
Are we still missing one? ECDSA_SECP384r1_SHA384?

Reply:
Hmmm good point. Initially I didn't plan to add all the options, but then I did add all the TLS1.2 options. Might as well add the last TLS1.3 option!

Reply:
Added! Test:
```
[nix openssl-3.0] $ integ signature_algorithms > output2
[nix openssl-3.0] $ grep --count ecdsa_secp384r1_sha384 output2
36
```